homeassistant/components/androidtv/media_player.py

```diff
@@ -157,9 +157,9 @@ def service_adb_command(service):
             output = target_device.adb_command(cmd)
 
             # log the output if there is any
-            if output and (not isinstance(output, str) or output.strip()):
+            if output.strip():
                 _LOGGER.info("Output of command '%s' from '%s': %s",
-                             cmd, target_device.entity_id, repr(output))
+                             cmd, target_device.entity_id, output)
```

Comment:
If repr is really wanted, you can use `%r` as the format string, afaik.

Reply:
I think `%s` is fine. In fact, I don't think it makes any difference, since `output` will only be logged when it is a non-empty string.